### PR TITLE
E_CLASSROOM-279 [FE/BE] Implement the add/edit of a question

### PR DESCRIPTION
**Pull request by CaneteJoash**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-279
### Definition of Done
- [x] User can edit the text of the question
- [x] User can select a different question type
- [x] Newly added question should be appended at the bottom
- [x] Newly added/edited questions should be shown in the question overview
### Commands to Run
N/A

### Related PR
be: https://github.com/framgia/sph-classroom-els-be/pull/73
#### Main note
- N/A
#### Pages Affected
- http://localhost:3003/admin/quizzes/1/edit

### Scenarios/ Test Cases
- [x] User can edit the text of the question
- [x] User can select a different question type
- [x] Newly added question should be appended at the bottom
- [x] Newly added/edited questions should be shown in the question overview
#### User Interface
N/A
#### User Experience 
N/A
#### Functionality
N/A
### Screenshots

adding a new Question
image
image

QuestionList

image

editing the Question
image
image
image

QuestionList
image

### CODE REVIEW

**Comment by catampongan** [comment]
## Working
- [x] Able to add question
- [x] Able to save questions
- [x] Able to add/remove choices on multiple choice type
- [x] Able to select different question type
- [x] Able to edit text of the questions
- [x] Added/edited questions can be shown in the question overview

## Comment
- [ ] Add spinner on the `add question` button if data is not rendered yet to avoid an error
- [ ] Put message "No question found" if there's no question on that specific quiz yet
- [ ] Move question type and time limit to the right side

------------------------------
Task-279

Task-279-1

Task-279-2

Task-279-3

-----------------------
- Total Test Time: 1 hr

File: src/pages/admin/quizzes/QuizEdit/index.module.scss [hunk 1]
```diff
@@ -236,17 +262,38 @@
...
-  display: inline-block;
+.radioStyle{
+  margin-top: 9px;
+  height: 30px;
+  width: 30px;
 }
+
+.questionTypeChoiceStyle{
+  margin-top: 500px;
+}
```
Comment (karlolee50) [inline src/pages/admin/quizzes/QuizEdit/index.module.scss]: put extra line

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 15 minutes
Comment/s: 1

**Comment by catampongan** [comment]
## Approved PR #118 
- [x] Able to add question
- [x] Able to save questions
- [x] Able to add/remove choices on multiple choice type
- [x] Able to select different question type
- [x] Able to edit text of the questions
- [x] Added/edited questions can be shown in the question overview
-----------------------------------
Task-279-1

Task-279-2

Task-279-3

-------------------------
- Total Test Time: 1 hr
- Total Iterations: 1
- Total Comments: 3

**Pull request merged**